src/app/kbv/controllers/self-assessment-router.js

```diff
@@ -0,0 +1,42 @@
+const path = require("path");
+const BaseController = require("hmpo-form-wizard").Controller;
+const presenters = require("../../../presenters");
+
+class SelfAssessmentRouterController extends BaseController {
+  configure(req, res, next) {
+    req.form.options.template = path.join(
+      req.form.options.templatePath,
+      "self-assessment-router"
+    );
+    super.configure(req, res, next);
+  }
+
+  locals(req, res, callback) {
+    super.locals(req, res, (err, locals) => {
+      if (err) {
+        return callback(err, locals);
+      }
+
+      const translationKey = "self-assessment-router";
```

Comment:
There is not a lot of unique content on this page, and the `SA100` / `SA200` is determined by `questionKey: sa-income-from-pensions` & `currentTaxYear": "2022/23`.

What do the other 2022/2023 questions like this look like?

Reply:
Current presenters use the questionKey to get the translations however in this scenario its not using the questionKey as thats used for the actual question pages to follow, this is just the router page. I could amend the presenter to pass in a custom translationKey but then would/should apply that change to all presenters.

Date specific currentTaxYear (eg, 2022/23), will be used inside inset fields on other question pages, not only titles like this case.